Doc/library/unittest.mock-examples.rst

```diff
@@ -1144,6 +1143,15 @@ the ``mock_calls`` attribute on the manager mock:
     >>> manager.mock_calls == expected_calls
     True
 
+However, parameters to calls that return mocks are not recorded, which means tracking
+nested calls where the parameters used to create ancestors are important is not possible:
```

Comment:
Note this is possible, you just need to use `call_list()`, as it was specified in the docs.

Reply:
Not across mocks it's not.

Reply:
What do you mean with across mocks?

Reply:
Will cover this in a non-in-line response...